### DISCUSSION TITLE
TypeError: Cannot redefine property: ethereum

### DISCUSSION THREAD

**Discussion by NXT-Zen** [phantom/docs#106] [Help & Support] [answered]
### Summary

![image](https://github.com/phantom/docs/assets/123107411/490129dc-63a9-457b-a0e9-cac9b09292d9)
When visiting a d-app for Solana, with Networks Enabled set as All, this error appears.

If I set network to solana only, this doesn't happen. 

### Steps to Reproduce

npm run dev 

Anybody else having this issue on their dapps? 

### Phantom Version

23.12.0

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
Hey @NXT-Zen,

Would you mind upgrading to 23.13.0 and confirming if the issue persists, please?

If it persists, do you have a hosted application where I can reproduce this behavior?

**Comment by NXT-Zen** [comment]
The issue persists.. I actually don't think it's only happening on my app, if I open the console I can see the error on almost any site that I visit. For example on this github page even.
<img width="487" alt="image" src="https://github.com/phantom/docs/assets/123107411/30a6284e-2602-4012-a8d0-f1a9a85d3d98">

**Reply by NXT-Zen** [reply]
@adamdelphantom

**Reply by wengzilla** [reply]
The most recent version of Phantom is _extremely_ buggy. I've been recommending all of my users to uninstall it for the time being. See this bug for something similar: https://github.com/orgs/phantom/discussions/103

**Reply by adamdelphantom** [reply]
@NXT-Zen I'm not able to reproduce this error. I tried a number of apps and I don't see the same error in the console. Would you mind pointing me to specific app where you're seeing this and also noting the browser and OS you're using, please?

**Reply by NXT-Zen** [reply]
Hmm.. 

It shows up on all consoles of any site I visit. I don't think it's app-specific. If you cannot see it, then it means it's to do with the rest of my system.

injected.js:2 Uncaught TypeError: Cannot redefine property: ethereum
    at Function.defineProperty (<anonymous>)
    at initEthereum (injected.js:2:3085632)
    at main (injected.js:2:3084392)
    at injected.js:2:3086206
    at injected.js:2:3086214

Browser: Chrome Version 116.0.5845.187 (Official Build) (arm64)
System: Mac OS 13.5.2 (22G91)

**Reply by NXT-Zen** [reply]
@adamdelphantom The good thing is that at least in production the error only appears in the console and does not actually mess up anything. Still, it's an uncaught error that I cannot fix in the app.

**Reply by adamdelphantom** [reply]
Do you have other wallet extensions installed? If, so which ones?

**Reply by NXT-Zen** [reply]
Metamask, Glow, Solflare and Coinbase wallet

**Reply by NXT-Zen** [reply, marked as answer]
Seems like it was caused by Backpack. I have uninstalled it and the issue is resolved.
